### PR TITLE
Add Remaining Tour Steps

### LINKED ISSUE #1174: Add missing steps for guided tour
Missing pages:
- [x] OAuth Settings
- [x] Dashboard
- [ ] Dashboard: Non-Empty State
- [x] Create Connection
- [x] Create Connection: Review
- [x] Create Integration
- [ ] Create Integration: Data Mapper
- [x] Create Integration: Review
- [x] Integration Detail Page

Improvements:
- [x] Style previous/next buttons to look like PF guided tour
- [x] Manual trigger of tour
- [x] Check for ID per @gashcrumb 's suggestion, so that tour does not show for existing integrations
- [ ] Check for step in URL for Create Integration, so that tour is not duplicated for second connection configuration flow

Document with text: https://docs.google.com/document/d/1tTgg4r6Wsa5dJwdifpGRptXzSCyOPxzBOVMWKiZBCAM/edit#
Slides with placement: https://docs.google.com/presentation/d/1C7L0acXdA-kjNdl22Bh8HZnof6RaiFpmpugC4qOc72U/edit

### PR DESCRIPTION
**Pull request by kahboom**
Update styling for Prev/Next to look more like PF guided tour.

fix #1174

### CODE REVIEW

**Comment by gashcrumb** [comment]
teh lint:

```

src/app/integrations/edit-page/action-configure/action-configure.component.ts[49, 47]: Unnecessary trailing comma
src/app/integrations/edit-page/action-configure/action-configure.component.ts[49, 48]: missing whitespace
src/app/integrations/edit-page/connection-select/connection-select.component.ts[40, 47]: Unnecessary trailing comma
src/app/integrations/edit-page/connection-select/connection-select.component.ts[40, 48]: missing whitespace
src/app/integrations/edit-page/integration-basics/integration-basics.component.ts[20, 47]: Unnecessary trailing comma
src/app/integrations/edit-page/integration-basics/integration-basics.component.ts[20, 48]: missing whitespace
```

**Comment by gashcrumb** [comment]
also if it's not WIP then you'll want to take off the WIP, otherwise the bot won't merge this PR when approved.

**Comment by kahboom** [comment]
Ready for review @gashcrumb . Thanks! 👍

File: src/app/app.component.html [hunk 1]
```diff
@@ -9,21 +9,9 @@
...
-          <span class="pficon pficon-close"></span>
-        </button>
-        <!--
-        <button (click)="tourService.end()"
-                class="close end-tour">
-          <span class="pficon pficon-close"></span>
-        </button>
-        -->
+      <div class="tour-step-navigation text-right">
+        <button *ngIf="tourService.hasPrev(step)" class="btn btn-sm btn-primary" (click)="tourService.prev()">« Prev</button>
```
Comment (gashcrumb) [inline src/app/app.component.html]: Should these attributes be split out into separate lines?

File: src/app/app.component.html [hunk 2]
```diff
@@ -9,21 +9,9 @@
...
+        <button *ngIf="tourService.hasNext(step)" class="btn btn-sm btn-primary" (click)="tourService.next()">Next »</button>
```
Comment (gashcrumb) [inline src/app/app.component.html]: Same as above

File: src/app/connections/create-page/review/review.component.ts [hunk 3]
```diff
@@ -13,12 +14,14 @@ import { CanComponentDeactivate } from '../../../common/can-deactivate-guard.ser
...
 import { ConnectionService } from '../../../store/connection/connection.service';
 import { log, getCategory } from '../../../logging';
+import { UserService } from '../../../common/user.service';
 
 const category = getCategory('Connections');
 
 @Component({
   selector: 'syndesis-connections-review',
   templateUrl: 'review.component.html',
+  styles: [ '' ],
```
Comment (gashcrumb) [inline src/app/connections/create-page/review/review.component.ts]: I don't think this line is necessary

File: src/app/dashboard/dashboard.component.ts [hunk 4]
```diff
@@ -37,15 +37,23 @@ export class DashboardComponent implements OnInit {
...
     this.connectionStore.loadAll();
     this.integrationStore.loadAll();
+
     /**
      * If guided tour state is set to be shown (i.e. true), then show it for this page, otherwise don't.
      */
     if (this.userService.getTourState() === true) {
       this.tourService.initialize([ {
-          anchorId: 'dashboard.navigation',
+        route: 'dashboard',
```
Comment (gashcrumb) [inline src/app/dashboard/dashboard.component.ts]: tabs seem off here

File: src/app/dashboard/dashboard.component.ts [hunk 5]
```diff
@@ -37,15 +37,23 @@ export class DashboardComponent implements OnInit {
...
+        anchorId: 'dashboard.navigation',
+          title: 'Navigation',
           content: 'View integrations, connections or settings for applications that Fuse Ignite is registered with.',
           placement: 'right',
-          title: 'Navigation',
+        }, {
+        route: 'dashboard',
```
Comment (gashcrumb) [inline src/app/dashboard/dashboard.component.ts]: same here with this line and the one after

File: src/app/connections/create-page/review/review.component.ts (hunk 3 above)
Comment (deeleman) [reply src/app/connections/create-page/review/review.component.ts]: It's not, but will make its way to the transpiled app unless removed. As a rule of thumb, `styles` or `styleUrls` properties should be removed from the decorator hash if not in use.

BTW, the new linter will complain when arrays containing stuff with blank spaces around. Is everybody agree?

File: src/app/dashboard/dashboard.component.ts (hunk 4 above)
Comment (deeleman) [reply src/app/dashboard/dashboard.component.ts]: Once merged, the new linter will take care of raising an alert for this upon committing code :)

File: src/app/connections/create-page/review/review.component.ts (hunk 3 above)
Comment (kahboom) [reply src/app/connections/create-page/review/review.component.ts]: Not sure why this is here tbh, I must have been testing things out. Sorry.

Comment (kahboom) [reply src/app/connections/create-page/review/review.component.ts]: Sure @deeleman sounds good to me.

File: src/app/dashboard/dashboard.component.ts (hunk 5 above)
Comment (kahboom) [reply src/app/dashboard/dashboard.component.ts]: That's one of the things I wasn't sure about.

File: src/app/app.component.html (hunk 1 above)
Comment (kahboom) [reply src/app/app.component.html]: Yes

Comment (gashcrumb) [reply src/app/app.component.html]: You know, lemme get an issue open for HTML formatting, maybe we can make a pre-commit hook for something there too...

**Pull request merged**